crates/jstz_cli/src/main.rs

```diff
@@ -85,38 +83,37 @@ enum Command {
     Kv(kv::Command),
 }
 
-async fn exec(command: Command, cfg: &mut Config) -> Result<()> {
+async fn exec(command: Command) -> Result<()> {
     match command {
-        Command::Sandbox(sandbox_command) => sandbox::exec(cfg, sandbox_command),
-        Command::Bridge(bridge_command) => bridge::exec(bridge_command, cfg),
-        Command::Account(account_command) => account::exec(account_command, cfg).await,
+        Command::Sandbox(sandbox_command) => sandbox::exec(sandbox_command),
+        Command::Bridge(bridge_command) => bridge::exec(bridge_command),
+        Command::Account(account_command) => account::exec(account_command).await,
         Command::Deploy {
-            self_address,
-            function_code,
+            code,
             balance,
             name,
-        } => deploy::exec(self_address, function_code, balance, name, cfg).await,
+        } => deploy::exec(code, balance, name).await,
         Command::Run {
             url,
-            referrer,
             http_method,
             gas_limit,
             json_data,
-        } => run::exec(cfg, referrer, url, http_method, gas_limit, json_data).await,
-        Command::Repl { self_address } => repl::exec(self_address, cfg),
-        Command::Logs(logs) => logs::exec(logs, cfg).await,
-        Command::Login { alias } => account::login(alias, cfg),
-        Command::Logout {} => account::logout(cfg),
-        Command::WhoAmI {} => account::whoami(cfg),
-        Command::Kv(kv_command) => kv::exec(kv_command, cfg).await,
+        } => run::exec(url, http_method, gas_limit, json_data).await,
+        Command::Repl { account } => repl::exec(account),
+        Command::Logs(logs) => logs::exec(logs).await,
+        Command::Login { alias } => account::login(alias),
+        Command::Logout {} => account::logout(),
+        Command::WhoAmI {} => account::whoami(),
+        Command::Kv(kv_command) => kv::exec(kv_command).await,
```

Comment:
at some point I want to also reorder and categorize these so that more commonly used commands like deploy/run are on the top.

Reply:
👍 sgtm